core/server/data/migrations/versions/3.12/001-add-identity-permission.js

```diff
@@ -0,0 +1,54 @@
+const ObjectId = require('bson-objectid');
```

Comment:
was there a reason for going into triple digits `/3.12/001-....` with the migration file name?

Reply:
Only so there's less chance of collision!

Reply:
Renamed to 01 FYI

Reply:
Think this will have to go in as `02-` I think as this landed in the master - https://github.com/TryGhost/Ghost/commit/6a9b53fcade38f63ce08cdb40f75f8f25bb0a85d